extensions/wikia/Email/Controller/WelcomeController.class.php

```diff
@@ -24,41 +25,72 @@ protected function getSubject() {
 	public function body() {
 		$this->response->setData( [
 			'salutation' => $this->getSalutation(),
-			'summary' => $this->getMessage( 'emailext-welcome-summary' )->text(),
-			'extendedSummary' => $this->getMessage( 'emailext-welcome-summary-extended' )->text(),
+			'summary' => $this->createSummary(),
 			'details' => $this->getDetailsList(),
-			'contentFooterMessages' => [
-				$this->getMessage( 'emailext-welcome-footer-community' )->parse(),
-				$this->getMessage( 'emailext-welcome-footer-closing' )->text(),
-				$this->getMessage( 'emailext-emailconfirmation-community-team' )->text()
-			],
 			'hasContentFooterMessages' => true,
+			'contentFooterMessages' => $this->createContentFooterMessages(),
+			'signatureIcon' => ImageHelper::getFileUrl( 'Fandom-heart-color.png' ),
+			'signature' => $this->getMessage( 'emailext-emailconfirmation-community-team' )->text()
 		] );
 	}
 
+	private function createSummary() {
+		return join(' ', [
+			$this->getMessage( 'emailext-welcome-summary' )->text(),
+			$this->getMessage( 'emailext-welcome-summary-extended' )->text(),
+			$this->getMessage( 'emailext-welcome-summary-extended-tips' )->text()]);
+	}
+
+	private function createContentFooterMessages() {
+		return [
+			join(' ', [
+				$this->getMessage( 'emailext-welcome-footer-community' )->parse(),
+				$this->getMessage( 'emailext-welcome-footer-closing' )->text()])
+		];
+	}
+
 	protected function getDetailsList() {
 		$basicsUrl = $this->getMessage( 'emailext-welcome-basics-url' )->text();
 		return [
 			[
-				'iconSrc' => ImageHelper::getFileUrl( 'Create-profile.png' ),
+				'iconSrc' => ImageHelper::getFileUrl( 'CreateYourProfile.png' ),
 				'iconLink' => $this->getCurrentProfilePage(),
 				'detailsHeader' => $this->getMessage( 'emailext-welcome-profile-header' )->text(),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "detailsHeader" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)